tests/store/tracking/test_file_store.py

```diff
@@ -476,18 +487,28 @@ def test_delete_restore_experiment(self):
         exp_id = self.experiments[random_int(0, len(self.experiments) - 1)]
         exp_name = self.exp_data[exp_id]["name"]
 
+        exp1 = fs.get_experiment(exp_id)
+        time.sleep(0.05)
```

Comment:
```suggestion
        time.sleep(0.01)
```

Let's use `0.01` to run tests faster.